src/main/java/core/basesyntax/figure/FigureSupplier.java

```diff
@@ -0,0 +1,68 @@
+package core.basesyntax.figure;
+
+import core.basesyntax.figure.color.ColorSupplier;
+import core.basesyntax.figure.name.RandomName;
+import java.util.Random;
+
+public class FigureSupplier {
+    //Constant
+    private static final int BOUND_NUMBER = 10;
+    private static final int RADIUS_CIRCLE = 10;
+    //Object
+    private ColorSupplier colorSupplier = new ColorSupplier();
+    private RandomName randomName = new RandomName();
+    private Random random = new Random();
+    //Random variables
+    private String color = colorSupplier.getRandomColor();
+    private String nameFigure = randomName.getRandomName();

```

Comment:
You don't need this variable, cause you have a figure name in the method `getRandomFigure()` and you can use that value